ocaml/ocaml-ppxlib/Portfile

```diff
@@ -16,13 +16,14 @@ long_description    Ppxlib is the standard library for ppx rewriters and other \
                     programs that manipulate the in-memory reprensation of the \
                     OCaml parsetree.
 
-checksums           rmd160  67c22d030355d20b20316f3c2d2ff09b3df03ac6 \
-                    sha256  0723f1385e7baaecb89ab43cd11f03fe36663d6e2bd54bf889d518c8b5cc5f29 \
-                    size    582387
+checksums           rmd160  ba6a99bc6e0eff1e0e5a967e98ecfb66c3d71e2f \
+                    sha256  4c96bfd834f971d08b300a15ad185d86b6f971b09e296cb406348bdbb23cb09d \
+                    size    610383
 github.tarball_from archive
 
+conflicts           ocaml-migrate-parsetree
+
```

Comment:
Are you sure about this conflict? How would you use ppxlib *without* migrate-parsetree?

Reply:
@pmetzger 
Opam website stated it. No, I am not sure. Could you please advise on this?

Reply:
Could you point me at the link so I can read it?

Reply:
https://opam.ocaml.org/packages/ppxlib

`ocaml-migrate-parsetree` is not in Dependencies now, but in Conflicts.

Do I misunderstand something here?

Reply:
Okay, so it appears that ppxlib now does all this stuff on its own, and ocaml-migrate-parsetree is deprecated and should be removed if nothing else directly depends on it; it will not run under OCaml 5.1 or successors at all.